gpii/node_modules/flowManager/src/PCPInterface.js

```diff
@@ -0,0 +1,132 @@
+/**
+* GPII PCP Render Handler
+*
+* Copyright 2014 Astea
+* Copyright 2015 Raising the Floor - International
+*
+* Licensed under the New BSD license. You may not use this file except in
+* compliance with this License.
+*
+* You may obtain a copy of the License at
+* https://github.com/gpii/universal/LICENSE.txt
+*/
+(function () {
+    "use strict";
+
+    var fluid = require("infusion");
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.request.flowManager");
+    fluid.registerNamespace("gpii.pcpInterface");
+
+    fluid.defaults("gpii.pcpInterface", {
+        gradeNames: ["autoInit", "fluid.eventedComponent"],
+        members: {
```

Comment:
Why do we imagine that there is exactly one socket? This seems like an invitation to trouble. We should instead just reuse Javi's existing "browserChannel" scheme (perhaps after suitable renaming and refactoring).